src/components/CreatePost.tsx

```diff
@@ -51,14 +53,14 @@ const CreatePost: React.FC<Props> = props => {
       [event.target.name]: event.target.value
     }));
     validatePost(event.target.value);
+    isAllValid();
```

Comment:
These lines are the root for the button problem you were telling me. Because inside of `validatePost` you are setting new state but the new state is not available for `isAllValid` to check for new values of the state. I would solve it by removing whole `formValid`
![image](https://user-images.githubusercontent.com/12229968/63964517-f6c18800-ca9f-11e9-8989-ad23fcaa751c.png)

And then here 
![image](https://user-images.githubusercontent.com/12229968/63964535-fde89600-ca9f-11e9-9438-53b6434b3c79.png)